app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+
+    if split_command[0] != "mv":
+        return
```

Comment:
it is better to print some informative message before that, so that the user understands what went wrong